src/endpoints/endpoints.services.module.ts

```diff
@@ -24,6 +24,7 @@ import { TransactionModule } from "./transactions/transaction.module";
 import { UsernameModule } from "./usernames/username.module";
 import { VmQueryModule } from "./vm.query/vm.query.module";
 import { WaitingListModule } from "./waiting-list/waiting.list.module";
+import {QueueWorkerModule} from "../queue.worker/queue.worker.module";
```

Comment:
QueueWorkedModule shouldn't be included in endpoints, this module will run on another instance

Reply:
Fixed